app/src/main/res/layout/activity_movie_details.xml

```diff
@@ -0,0 +1,331 @@
+            app:srcCompat="@drawable/shadow_bottom" />
+
+        <TextView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="18dp"
+            android:layout_marginTop="56dp"
+            android:drawablePadding="8dp"
+            android:text="@string/back_button"
+            android:textColor="@color/colorTextGray"
+            android:textSize="12sp"
+            app:drawableStartCompat="@drawable/arrow_back"
+            app:layout_constraintStart_toStartOf="@id/movie_poster"
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <TextView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginBottom="12dp"
+            android:text="@string/pg"
+            android:textColor="@android:color/white"
+            android:textSize="12sp"
+            android:textStyle="bold"
+            app:layout_constraintBottom_toTopOf="@id/movie_title"
+            app:layout_constraintStart_toStartOf="@id/movie_title" />
+
+        <TextView
+            android:id="@+id/movie_title"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:text="@string/avengers_title"
+            android:textColor="@android:color/white"
+            android:textSize="40sp"
+            android:textStyle="bold"
+            app:layout_constraintHorizontal_bias="0"
+            app:layout_constraintStart_toEndOf="@id/left_guideline"
+            app:layout_constraintEnd_toStartOf="@id/right_guideline"
+            app:layout_constraintTop_toBottomOf="@id/movie_poster" />
+
+        <TextView
+            android:id="@+id/tag_line"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="@string/tags"
+            android:textColor="@color/colorInk"
+            android:textSize="14sp"
+            android:layout_marginVertical="4dp"
+            app:layout_constraintHorizontal_bias="0"
+            app:layout_constraintStart_toEndOf="@id/left_guideline"
+            app:layout_constraintEnd_toStartOf="@id/right_guideline"
+            app:layout_constraintTop_toBottomOf="@id/movie_title" />
+
+        <TextView
+            android:id="@+id/reviews"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="@string/reviews"
+            android:textColor="@color/colorTextGray"
+            android:textSize="14sp"
+            android:layout_marginVertical="4dp"
+            android:layout_marginStart="9dp"
+            app:layout_constraintStart_toEndOf="@id/star5"
+            app:layout_constraintEnd_toStartOf="@id/right_guideline"
+            app:layout_constraintTop_toBottomOf="@id/tag_line" />
+
+        <androidx.appcompat.widget.AppCompatImageView
```

Comment:
You could use `ImageView` here, because for AppCompatActivity it  will use AppCompatImageView under the hood anyways.